docs/docs-content/downloads/artifacts-studio.md

```diff
@@ -0,0 +1,263 @@
+---
+sidebar_label: "Artifact Studio"
+title: "Artifact Studio"
+description: "Overview of the Artifact Studio"
+hide_table_of_contents: false
+sidebar_position: 10
+sidebar_custom_props:
+  icon: "layer-group"
+tags: ["downloads", "artifact-studio"]
+---
+
+[Artifact Studio](https://artifact-studio.spectrocloud.com/) is a unified platform that helps airgapped,
+regulatory-focused and security conscious organizations to populate their own internal registries with bundles, packs
+and installers to be used with Palette or VerteX. It provides a single location for packs and images, streamlining
+access and management.
+
+**Who is this for?**
+
+All self-hosted customers will be able to download the Palette or VerteX Management Appliance or Helm chart in order to
+instantiate or upgrade their environment. Additionally, disconnected customers are able to browse through all available
+packs, and download only the components you wish to use. You can also validate the integrity and authenticity of the
+bundle via code signature. With these new artifact bundles, we are replacing the self-extracting binaries we create for
+airgapped environments with a more familiar zstandard compressed format (`zst`) that can be moved and manipulated with
+palette-cli.
+
+There are four main artifact areas:
+
+    - Install Palette Enterprise (content bundle, ISO or Helm chart).
+    - Install Palette VerteX (content bundle, ISO or Helm chart).
+    - Create a pack bundle to download.
+    - Browse pack Catalog to download individual packs.
+
+:::preview :::
+
+![Image of the default homepage of Artifact Studio](../../../static/assets/docs/images/downloads/downloads_artifact-studio-main.webp)
+
+## Install Palette Enterprise
+
+1. Navigate to [Artifact Studio](https://artifact-studio.spectrocloud.com/).
+
+2. From the drop-down, select the version needed, and click **Show Artifacts**.
+
+3. Click **Download** on **Content bundle**, **Palette management appliance ISO**, or **Helm installation**, whichever
+   file is appropriate for your environment.
+
+Once you have the file, you can deploy Palette as a self-hosted application. Review the
+[Self-Hosted Palette](../enterprise-version/install-palette/install-palette.md) for more information on deploying
+Palette locally.
+
+## Install Palette VerteX
+
+1. Navigate to [Artifact Studio](https://artifact-studio.spectrocloud.com/).
+
+2. From the drop-down, select the version needed, and click **Show Artifacts**.
+
+3. Click **Download** on **Content bundle**, **VerteX management appliance ISO**, or **Helm installation**, whichever
+   file is appropriate for your environment.
+
+Once you have the file, you can deploy Palette VerteX as a self-hosted application. Review the
+[VerteX Installation](../vertex/install-palette-vertex/install-palette-vertex.md) for more information on deploying
```

Comment:
This will be `../vertex/install-palette-vertex/vertex-management-appliance.md` once both PRs are merged.